src/app/login/AccountRecoveryRequestForm.tsx

```diff
@@ -75,9 +75,12 @@ export default function AccountRecoveryRequestForm({
         open={openForm}
         onClose={onClose}
         data-testid="login-dialog-recover"
+        fullWidth={true}
+        maxWidth='md'
+        style={{  padding: '20px', borderRadius: '20px 20px 0 0' }}
```

Comment:
you can create a seperate component so it it not redundant and wrap the cildren in it